catalog/gitops/hydration-trigger.yaml

```diff
@@ -44,6 +44,9 @@ spec:
 
             echo "Executing Kpt Functions..."
             kpt fn render "$${SRC_DIR}" --output="$${DEST_DIR}" --truncate-output=false
+
+            echo "Removing local config"
+            kpt fn eval "$${DEST_DIR}" -i gcr.io/yakima-eap/remove-annotated-resources:v0.1
```

Comment:
tested this locally with a local-config annotation and seems to WAI. Is this checked in yet?